src/screens/Dashboard/DashboardScreen.tsx

```diff
@@ -271,6 +284,8 @@ const mapStateToProps = (state: ApplicationState) => ({
   allTransactions: walletsSelectors.transactions(state),
   transactionNotes: transactionsNotesSelectors.transactionNotes(state),
   isFilteringOn: state.filters.isFilteringOn,
+  airdropThankYouFlowCompleted: airdropSelectors.thankYouFlowCompleted(state),
+  airdropThankYouSeen: airdropSelectors.thankYouSeen(state),
```

Comment:
not used

Reply:
as above, I ve left it because we will use it soon, but yeah, maybe better remove it after all.

Reply:
done